rocAL/rocAL/source/coco_meta_data_reader.cpp

```diff
@@ -80,16 +80,16 @@ void COCOMetaDataReader::print_map_contents()
 {
     BoundingBoxCords bb_coords;
     BoundingBoxLabels bb_labels;
-    ImgSizes img_sizes;
+    ImgSize img_size;
 
     std::cout << "\nBBox Annotations List: \n";
     for (auto &elem : _map_content)
     {
         std::cout << "\nName :\t " << elem.first;
         bb_coords = elem.second->get_bb_cords();
         bb_labels = elem.second->get_bb_labels();
-        img_sizes = elem.second->get_img_sizes();
-        std::cout << "<wxh, num of bboxes>: " << img_sizes[0].w << " X " << img_sizes[0].h << " , " << bb_coords.size() << std::endl;
+        img_size = elem.second->get_img_size();
+        std::cout << "<wxh, num of bboxes>: " << img_size.w << " X " << img_size.h << " , " << bb_coords.size() << std::endl;
```

Comment:
remove print statement from active code

Reply:
This print statement is a part of print_map_contents() function